packages/alice_objectbox/lib/model/cached_alice_http_response.dart

```diff
@@ -0,0 +1,60 @@
+import 'dart:convert' show jsonDecode, jsonEncode;
+
+import 'package:alice/model/alice_http_response.dart';
+import 'package:objectbox/objectbox.dart';
+
+@Entity()
+class CachedAliceHttpResponse implements AliceHttpResponse {
+  CachedAliceHttpResponse({
+    this.objectId = 0,
+    this.status = 0,
+    this.size = 0,
+    DateTime? time,
+    this.body,
+    this.headers,
+  }) : time = time ?? DateTime.now();
+
+  @Id()
+  int objectId;
```

Comment:
Shouldn't fields be final?

Reply:
Yes they can, except this one
```dart
@Id()
int objectId;
```
as it is an ObjectBox requirement